apps/OpenFOAM/ARCHER2-OpenFOAM-v6.20180710/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenFOAM v6
+
+This is OpenFOAM version 6
+See https://openfoam.org/download/6-linux/
+
+This directory contains a subdirectory `site` with the
+following scripts which allow one to install and compile OpenFOAM
+using the Gnu compiler collection on Archer2.
```

Comment:
Can you make sure ARCHER2 has the correct capitalisation throughout? Sounds petty, but as the service provider we need to get it right

Reply:
Not doing very well on this... need a script!